.github/workflows/docker-deploy.yml

```diff
@@ -16,18 +16,32 @@ on:
     - cron: "3 17 * * 4"
 
 jobs:
+  date:
+    runs-on: ubuntu-latest
+    outputs:
+      date: ${{ steps.date.outputs.date }}
+      snapshot_date: ${{ steps.date.outputs.snapshot_date }}
+    steps:
+      - name: Get date
+        id: date
+        run: |
+          export SNAPSHOT_DATE=$(basename $(curl -ILs -o /dev/null -w %{url_effective} http://snapshot.debian.org/archive/debian/$(date -u +%Y%m%dT%H%M00Z)/) )
+          echo "snapshot_date=${SNAPSHOT_DATE}" >> $GITHUB_OUTPUT
+          echo "date=$(date '+%Y_%m_%d')" >> $GITHUB_OUTPUT
+
   # There is unfortunately no point in parallelising the build of the different
   # images, because they depend on each other. So sequential is the best we can do.
+  # We still split of the l4v build, because the GitHub runner otherwise runs out of
+  # disk space.
   build:
     name: Docker
     runs-on: ubuntu-latest
+    needs: date
+    env:
```

Comment:
maybe drop this and section and use `${{ needs.date.outputs.date }}` directly?

Reply:
I was initially going for that, but found it confusing to  mix the JS `${{ .. }}` into the shell commands. This way, the shell commands are pure shell, without any special JS in the middle.

Reply:
You have a point there. I was also thinking some time about this was is better. It's something to keep in mind, that variables are not passed trivially across jobs.